src/components/StageEditor/sections/CategoricalBinPrompts/CategoricalBinPrompts.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Prompts from '../../../Prompts';
+import CategoricalBinPrompt from './CategoricalBinPrompt';
+
+const CategoricalBinPrompts = props => (
+  <Prompts
+    contentId="guidance.editor.name_generator_list_prompts"
+    promptComponent={CategoricalBinPrompt}
+    template={{ showExistingNodes: true }}
```

Comment:
I don't see `showExistingNodes` supported in NC. Do we need a ticket to implement there?

Reply:
That's a mistake, it's now been removed